api/src/database/system-data/fields/settings.yaml

```diff
@@ -181,6 +181,134 @@ fields:
+                        - value: resize
+                          text: resize
+                        - value: extend
+                          text: extend
+                        - value: extract
+                          text: extract
+                        - value: trim
+                          text: trim
+                        - value: rotate
+                          text: rotate
+                        - value: flip
+                          text: flip
+                        - value: flop
+                          text: flop
+                        - value: sharpen
+                          text: sharpen
+                        - value: median
+                          text: median
+                        - value: blur
+                          text: blur
+                        - value: flatten
+                          text: flatten
+                        - value: gamma
+                          text: gamma
+                        - value: negate
+                          text: negate
+                        - value: normalize
+                          text: normalize
+                        - value: clahe
+                          text: clahe
+                        - value: convolve
+                          text: convolve
+                        - value: threshold
+                          text: threshold
+                        - value: linear
+                          text: linear
+                        - value: recomb
+                          text: recomb
+                        - value: modulate
+                          text: modulate
+                        - value: tint
+                          text: tint
+                        - value: greyscale
+                          text: greyscale
+                        - value: toColorspace
+                          text: toColorspace
+                        - value: removeAlpha
+                          text: removeAlpha
+                        - value: ensureAlpha
+                          text: ensureAlpha
+                        - value: extractChannel
+                          text: extractChannel
+                        - value: bandbool
+                          text: bandbool
+                  width: half
+                - field: arguments
+                  name: Arguments
+                  type: json
+                  meta:
+                    interface: list
+                    note: A list of arguments to pass to the Sharp method.
+                    options:
+                      template: '{{argument}}'
+                      fields:
+                        - field: argument
+                          name: Argument
+                          type: string
+                          schema:
+                            is_nullable: false
+                          meta:
+                            interface: input-code
+                            note: JSON representation of the argument
```

Comment:
We should figure out how to document this, little difficult to understand how to use it